code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -404,6 +404,13 @@ GLOBAL_LIST_INIT(maintenance_loot, list(
 	GLOB.maint_fauna = maint_fauna_weight,
 	))
 
+//Monke, loot pool of uncommon or better loot, used in maint god rituals.
```

Comment:
please move this to a modular directory

Reply:
What folder/file should I put this in? There is no globalvar directory in the monkestation folder. Should I make a globalvar directory then or put it with the ritual folder?